.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,18 @@
+name: spellcheck 
+on:
+  push:
+    paths:
+       - '**.asciidoc'
+       - '**.md'
```

Comment:
if we also support markdown we should IMHO also support `**.adoc` (see e.g. our README.adoc).